src/main/features/dashboard/claims-state-machine/full-defence-transitions.ts

```diff
@@ -0,0 +1,172 @@
+
+import * as StateMachine from '@taoqf/javascript-state-machine'
+import { Claim } from 'claims/models/claim'
+
+import { FreeMediationOption } from 'forms/models/freeMediation'
+import { DefenceType } from 'claims/models/response/defenceType'
+import { ClaimantResponseType } from 'claims/models/claimant-response/claimantResponseType'
+import * as _ from 'lodash'
+import * as path from 'path'
+import { FullDefenceStates } from 'claims/models/claim-states/full-defence-states'
+import { FullDefenceResponse } from 'claims/models/response/fullDefenceResponse'
+
+export function fullDefenceTransitions (claim: Claim): StateMachine {
```

Comment:
Please reformat this file to make it consistent.  There are few places with wrong spaces.